runtime/Jakt/StringBuilder.h

```diff
@@ -153,6 +153,23 @@ struct Formatter<JaktInternal::Set<T>> : Formatter<StringView> {
     }
 };
 
+template<typename T>
+struct Formatter<JaktInternal::Range<T>> : Formatter<StringView> {
+    ErrorOr<void> format(FormatBuilder& builder, JaktInternal::Range<T> const& range)
+    {
+        if (m_alternative_form)
+            JaktInternal::_pretty_print_enabled = true;
```

Comment:
Who sets this back to its original value?

Reply:
Yeah good catch, seems it wasn't reset in any cases, I had just copied it across from the array formatter.
Added a commit to fix it in runtime and codegen.